terraform/environments/ci/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+    backend "s3" {
+        bucket = "hashbang-terraform-ci"
+        key = "state.tfstate"
+        region = "us-west-2"
+        lock_table = "hashbang-terraform-ci"
+    }
+}
+
+provider "aws" {
+    region = "us-west-2"
+}
+
+module "vpc" {
+    source = "github.com/terraform-community-modules/tf_aws_vpc?ref=v1.0.6"
```

Comment:
Does that mean that anyone with push access to `github.com/terraform-community-modules/tf_aws_vpc` gets code execution as the admin running terraform (or as our CD infra, eventually)?